oregon/templates/site.html

```diff
@@ -38,10 +38,16 @@
         margin-top: 10px;
     }
     .place-label {
-    fill:#8C001A;
-    font-size:14;
-    font-weight: bold;
-    text-decoration:none !important;
+        fill:#951916;
```

Comment:
Next time put these kinds of changes in their own commit - they aren't related to adding the JS files